### PR TITLE
Chore/docs 502/icon sizing

### PR DESCRIPTION
**Pull request by stef-cobaltio**
Updating the icons on the homepage to be sized smaller.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Review by mtsuji-cobalt — COMMENTED** [review]
Screenshot 2024-11-12 at 12 13 53

i'm not sure we need the images here. if we do, we could add more padding between the image and text below and/or place the image to the left (not sure how this would look)

File: content/en/_index.md [hunk 1]
```diff
@@ -48,48 +48,44 @@ cascade:
...
-        <div class="h-100 p-5 border rounded-lg" style="border-color: #D6E3FD">
-          <h3>Manage Your In-House Pentests</h3>
-          <p class="my-2">{{% beta-label %}}</p>
-          <p>Increase the efficiency and quality of your in-house pentests with the Cobalt Pentest Management Platform. Manage all your pentests in one place to streamline your end-to-end processes and standardize reporting.</p>
-          <a href="/pmp/">Learn more »</a>
-        </div>
+  <div class="col-md-6">
+    <div class="card">
+      <div class="card-content">
+  <h3><b class="title-color">Try our DAST Scanner</b></h3>
```
Comment (mtsuji-cobalt) [inline content/en/_index.md:74]: This is nitpicking - if we're using sentence case here, we should apply that to 'Try engagements' or vice versa with title case

**Review by mtsuji-cobalt — COMMENTED** [review]
​just noticed that throughout our docs, text in blue are used to indicate clickable links. but on the landing page, the 6 categories are in black yet clickable while the titles of the 2 cards below are in blue but are not clickable. not sure how it'd look but we could swap the colors to be consistent

File: content/en/_index.md (hunk 1 above)
Comment (stef-cobaltio) [reply content/en/_index.md:74]: I am using title case here though, not sentence case, same as the section blocks above.

Comment (mtsuji-cobalt) [reply content/en/_index.md:74]: Oh, i was thinking 'our' needs to be capitalized but that doesn't seem to be the case

**Review by mtsuji-cobalt — APPROVED** [review]

File: content/en/_index.md (hunk 1 above)
Comment (stef-cobaltio) [reply content/en/_index.md:74]: I just used a converter, you are right - changing real quick.

**Pull request merged**